### ISSUE TITLE
Question about the result of ReAct-ResNet

### ISSUE THREAD

**Issue by ThisisBillhe** [liuzechun/ReActNet#29]
Hi zechun, 
Thanks for your excellent work! I'm trying to reproduce your result of ReAct-ResNet without changing any code. But I only get a result of 63.2%, which is far away from your official result (65.9% in README). I noticed that the official code training network for 128 epochs  and you commented the line of setting number of  epochs to 256. Is this the reason?  Or what else would affect  the result?

Looking forward to your reply!!

**Comment by liuzechun** [comment]
Hi, Thanks for your interest in our work! Using the new recipe in run.sh file should give you the same accuracy as 256 epochs. Did you follow the learning rate and batchsize in the run.sh?
